packages/cli/src/lib/run/dyno.ts

```diff
@@ -312,14 +312,28 @@ export default class Dyno extends Duplex {
         sshProc.stdout.on('data', this._readData())
       }
 
-      // @ts-ignore
-      sshProc.stderr.on('data', data => {
-        lastErr = data
+      sshProc.stderr?.on('data', data => {
+        lastErr = data.toString()
 
         // suppress host key and permission denied messages
-        if (this._isDebug() || (data.includes("Warning: Permanently added '[127.0.0.1]") && data.includes('Permission denied (publickey).'))) {
+        const messages = [
+          "Warning: Permanently added '[127.0.0.1]"
+        ]
+
+        const killMessages = [
+          'too many authentication failures',
```

Comment:
should we move these strings into consts so that there's no chance they change in one place and not the other? seems like a copy change to errors could break this, maybe non-issue

Reply:
This entire file could use a modernization uplift. I'll add a backlog item to discuss this with the team.